src/containers/MyNdla/Folders/FolderActions.tsx

```diff
@@ -240,6 +265,10 @@ const FolderActions = ({ selectedFolder, setFocusId, folders, inToolbar = false,
       return actions.concat(editFolder, deleteOpt);
     }
 
+    if (selectedFolder.__typename !== "Folder") {
+      return actions.concat(deleteLink);
+    }
```

Comment:
Du må endre litt her. Testa å logge inn som en elev og lagra ei lenke til mappe. Etterpå får eg disse valga:
![image](https://github.com/NDLANO/ndla-frontend/assets/8956884/5f5e12f6-93ca-4a2c-b527-b1a0cb9ed57e)
Det er valget rett over som blander seg. Her må du skille mellom valg for mapper og lenker til delte mapper.